private-templates-service/client/src/components/Common/PublishModal/styled.tsx

```diff
@@ -25,23 +25,27 @@ export const Modal = styled.div`
 `;
 
 export const Header = styled.div`
-  font-size: 1.375rem;
+  font-family: ${FONTS.SEGOE_UI_SEMI_BOLD};
+  font-size: 1.75rem;
   font-weight: 500;
   margin-bottom: 12px;
 `;
 
 export const Description = styled.div`
+  font-family: ${FONTS.SEGOE_UI_REGULAR};
```

Comment:
Nit: Do we need to explicitly specify the SEGOE_UI_REGULAR font? I think it's just inherited from the body?

Reply:
Hmm wasn't sure, I can remove it if it's unnecessary

Reply:
Does the font look different if you remove it? If not, I would say remove it